config-vm.md

```diff
@@ -61,8 +61,54 @@ This image contains the root filesystem that the virtual machine **`kernel`** wi
     }
 ```
 
+## <a name="HwConfigObject" /> HWConfig Object
+
+**`hwConfig`** (object OPTIONAL) Specifies the hardware configuration that should be passed to the VM.
+* **`deviceTree`** (string OPTIONAL) Path to the container device-tree file that should be passed to the VM.
+* **`vcpus`** (int OPTIONAL) Number of virtual cpus for the VM.
+* **`memory`** (int OPTIONAL) Maximum memory in bytes allocated to the VM.
+* **`dtdevs`** (array OPTIONAL) Host device tree nodes to passthrough to the VM, see [Xen Config][xl-config-format] for the details.
```

Comment:
There's several references here to Xen specifically -- are these all applicable to _other_ implementations too, or are these mostly Xen-specific fields?

I'm trying to understand how this helps improve interoperability between runtimes, so another way to put this question is whether you envision more than one runtime will implement support for this?

Reply:
This configuration is applicable for xen and stores information about the devices that should be passed to xen guest. That's why I marked it as OPTIONAL. I believe that there are similar configurations in other Type-1 hypervisors but I think that each implementation has it's specific format and additional fields. Some staff like "vcpus", "memory" and so on are generic, but hardware passthrough fields are to specific to the exact implementation and even architecture. I don't see the generic way to define this fields right now.